### PR TITLE
solved the merge conflict

### PR DESCRIPTION
**Pull request by Frankdevlopes**
removed some fields in the marker pop up , styled it and modified the map to center the pop up to prevent it from been shadowed by other elements the search bar when the marker is clicked.

### CODE REVIEW

File: src/components/shared/map/MapMarkerPopup.tsx [hunk 1]
```diff
@@ -1,65 +1,86 @@
...
+        <h2 className="font-bold text-sm text-center">{seller.name}</h2>
 
-        <div className="flex space-x-4 gap-2">
-          <Link
-            href={`/seller/sale-items/${seller.seller_id}`} //change to items list screen
-            className="flex-1">
-              <Button label={t('SHARED.BUY')} styles={{color: '#ffc153', paddingTop: '6px', paddingBottom: '6px', width: '100%' }} />
-          </Link>
-        </div>
+        {/* Close (X) button */}
```
Comment (swoocn) [inline src/components/shared/map/MapMarkerPopup.tsx]: You can leverage the `CloseButton` component that is in our **shared/Forms/Buttons** directory.

Comment (Frankdevlopes) [reply src/components/shared/map/MapMarkerPopup.tsx]: okay on it.

**Comment by swoocn** [comment]
Great initial effort, @Frankdevlopes! 👍
I have some main comments regarding the design alignment between your implemented design and the requirements outlined in the task reference @ [https://sharing.clickup.com/9015444408/t/h/86bz96zd5/AG5Q3PYG9FDSMYQ](https://sharing.clickup.com/9015444408/t/h/86bz96zd5/AG5Q3PYG9FDSMYQ).

Please review the key points below ->

image

1. Keep the zoom level consistent rather than zooming into the exact location of the shop marker.  We want users to still see the surrounding shop markers in relative to their opened shop marker to maintain their reference point.  Also, please ensure the shop marker remains displayed beneath the popup.
2. Move the close button to the top-right corner of the popup.
3. Center the shop name.
4. Please ensure you are using the translated version of the seller types.  You can refer to `seller/registration/page.tsx` for the implementation.
5. Position the Trust-o-meter bar just below the static Trust-o-meter text, as shown in the expected screenshot.
6. Update the 'Buy' button color scheme to match the requirements (yellow text on a green background); this is a shared button component.
7. Perhaps the screenshot I took of your design cropped out the bottom buttons (+Sell and Find Me), but please make sure these two buttons are still present at the bottom.

Thanks again for your efforts, and feel free to reach out if you have any questions!

**Review by swoocn — CHANGES_REQUESTED** [review]
Please make sure your implementation matches the requirements as closely as possible.
Thanks! 👍

**Comment by swoocn** [comment]
This PR is superseded by [https://github.com/map-of-pi/map-of-pi-frontend-react/pull/248](https://github.com/map-of-pi/map-of-pi-frontend-react/pull/248).

**Pull request closed without merging**